apps/app/app/globals.css

```
@@ -2,285 +2,226 @@
+     * always-mounted capture bar. Without this, tabbing to a control near
+     * either edge scrolls it flush against the viewport and straight under
+     * one of them, so the focus ring is invisible and a keyboard user cannot
+     * tell where they are. The values clear each bar with room to spare.
+     */
+    scroll-padding-top: 4.5rem;
+    scroll-padding-bottom: 13rem;
+  }
+
   body {
     @apply bg-background text-foreground;
+    font-size: 0.9375rem;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)